exercises/practice/etl/etl_test.sql

```diff
@@ -0,0 +1,29 @@
+-- Setup test table and read in student solution:
+.read ./test_setup.sql
+
+-- Test cases:
+INSERT INTO tests (name, uuid,
+                    input, expected)
```

Comment:
Is this indentation intentional?

Reply:
This is also consistent with the rest. It can defeinitely be changed, but I think that's a different PR.